lib/enumerable.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+module Enumerable
+  # my_each method
+  def my_each
+      i = 0
+      until i == self.length
+        yield (self[i])
+        i += 1
+      end
+  end
+
+  #my_each_with_index method
+  def my_each_with_index
+    i = 0
+    until i == self.length
+      yield(self[i], i)
+      i += 1
+    end
+  end
+
+  #my_count method
+  def my_count
+    cnt = 0
+    self.my_each{|ele|
+
+      if block_given?
+        cnt+=1 if yield(ele) != false
```

Comment:
It is good practise to have space around operator.
cnt += 1, will be better.....kindly check the rest of your code.